ui/app/components/app/connected-accounts-list/connected-accounts-list-options/connected-accounts-list-options.component.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types'
+import React, { PureComponent } from 'react'
+
+export default class ConnectedAccountsListOptions extends PureComponent {
+  static contextTypes = {
+    t: PropTypes.func.isRequired,
+  }
+
+  static propTypes = {
+    expanded: PropTypes.bool,
+    onClick: PropTypes.func,
+    children: PropTypes.node.isRequired,
+  }
+
+  static defaultProps = {
+    expanded: false,
+    onClick: undefined,
+  }
+
+  render () {
+    const { expanded, onClick, children } = this.props
+    const { t } = this.context
+
+    return (
+      <button className="fas fa-ellipsis-v" title={t('accountOptions')} onClick={onClick}>
+        {
+          expanded
+            ? (
+              <div className="connected-accounts-options">
```

Comment:
I noticed that this isn't being wrapped in a portal anywhere. This is probably fine, but maybe wrapping it in a Portal would be a good idea as a precaution, in case the options are ever wide or long enough to exceed the bounds of the modal this is rendered in.

Reply:
Do we have a reusable portal?

Reply:
No, I don't believe so. What would it do - create the DOM node to anchor the portal on? 🤔 There's not much to do aside from that.

Sounds like a good use case for a hook.